ui/apps/platform/src/Containers/Vulnerabilities/VulnerablityReporting/VulnReportingPage.tsx

```diff
@@ -0,0 +1,27 @@
+import React from 'react';
+import { Route, Switch } from 'react-router-dom';
+import { PageSection } from '@patternfly/react-core';
+
+import { vulnerabilityReportingPath } from 'routePaths';
+
+import PageNotFound from 'Components/PageNotFound';
+import PageTitle from 'Components/PageTitle';
+import VulnReportsPage from './VulnReports/VulnReportsPage';
+
+import './VulnReportingPage.css';
+
+function VulnReportingPage() {
+    return (
+        <Switch>
+            <Route exact path={vulnerabilityReportingPath} component={VulnReportsPage} />
+            <Route>
+                <PageSection variant="light">
+                    <PageTitle title="Vulnerability Reporting - Not Found" />
+                    <PageNotFound />
```

Comment:
nit: no need to bother adding the NotFound parts, when you know that will be replaced this sprint